solutions/59.js

```diff
@@ -19,6 +21,20 @@ const solution = (num) => {
   }
   return true;
 };
+
+const solution1 = (num) => {
+  if(num === 1 || num === 0 || num === -1){
+    return false;
+  }
+  for(let i = 2; i < Math.sqrt(num); i++){
```

Comment:
![screen shot 2017-06-15 at 10 43 52 am](https://user-images.githubusercontent.com/11692241/27194493-965771d6-51b7-11e7-9ddb-7d6ef7d2a996.png)

I solved it in my own way and my code passes, why not merge?